mix.exs

```diff
@@ -53,7 +53,8 @@ defmodule LambdaEthereumConsensus.MixProject do
       {:benchee, "~> 1.2", only: [:dev]},
       {:aja, "~> 0.6.2"},
       {:dialyxir, "~> 1.1", only: [:dev, :test], runtime: false},
-      {:credo, "~> 1.7", only: [:dev, :test], runtime: false}
+      {:credo, "~> 1.7", only: [:dev, :test], runtime: false},
+      {:crc32c, "~> 1.2"}
```

Comment:
Please change this to 

```elixir
      {:crc32c, git: "https://github.com/lambdaclass/crc32c", branch: "bump-rustler-to-29"}
```

Reply:
thanks a lot! I really appreciate your help 🙏🫡